backend/src/main/java/com/huellapositiva/application/controller/EmailConfirmationApiController.java

```diff
@@ -24,7 +26,11 @@
     @GetMapping("/{hash}")
     @ResponseStatus(HttpStatus.NO_CONTENT)
     public void confirmEmail(@PathVariable UUID hash) {
-        emailConfirmationAction.execute(hash);
+        try {
+            emailConfirmationAction.execute(hash);
+        }catch (EmailConfirmationExpired e){
```

Comment:
Don't need to catch the exception. It is already annotated with @ResponseStatus(HttpStatus.GONE)